climate_index_collection/output.py

```diff
@@ -0,0 +1,110 @@
+import xarray as xr
+import pandas as pd
+
+from pathlib import Path
+
+from climate_index_collection.indices import southern_annular_mode
+from climate_index_collection.data_loading import load_data_set
+
+
+def compute_index(data_path="../data/test_data/", data_source_name="FOCI", index_function=southern_annular_mode):
+    """Compute index from data source and return xarray DataArray.
+    
+    Parameters
+    ----------
+    data_path: str | path
+        Location of the data files. Defaults to "data/test_data/".
+    data_source_name: str
+        Name of the model dataset. Defaults to "FOCI".
+    index_function: function
+        Index function. Defaults to southern_annular_mode.
+        
+    Returns
+    -------
+    xarray DataArray
+    
+    """
+    data_set = load_data_set(data_path=data_path, data_source_name=data_source_name)
+    index_data_array = index_function(data_set)
+        
+    return index_data_array
+
+
+def index_dataarray_to_dataframe(index_data_array=None, data_source_name="FOCI"):
+    """Convert index from xarray DataArray to Pandas dataframe.
+    
+    Parameters
+    ----------
+    index_data_array: xarray DataArray
+        Defaults to None.
+    data_source_name: str
+        Name of the model dataset. Defaults to "FOCI".
+            
+    Returns
+    -------
+    Pandas dataframe
+    
+    """
+    index_df = index_data_array.to_dataframe().reset_index()
+    index_df['model']=data_source_name
```

Comment:
I'm not sure, but I think it would be better to set `data_source_name` as an attribute of the `index_data_array` in `compute_index()` already?